spec/paperclip/integration_spec.rb

```diff
@@ -262,6 +262,36 @@
     dummy.save
   end
 
+  [0777,0775,0750].each do |perms|
+    context "when the path perms are #{perms}" do
+      before do
+        FileUtils.rm_rf "tmp"
+        rebuild_model override_path_permissions: perms
+        @dummy = Dummy.new
+        @file  = File.new(fixture_file("5k.png"), 'rb')
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.